Author: Suruchi Agarwal <quic_suruchia@quicinc.com>
Date:      Thu Jan 13 08:24:18 2022 -0800
    nat46: MAP-T offload support

        Set IFF_EXT_MAPT flag in the priv_flags_ext of netdev on mapt device creation.

	Change-Id: I6b1ed3912cf53fadbd9abe8b2f9f693d7a748aab
	Signed-off-by: Suruchi Agarwal <quic_suruchia@quicinc.com>

	--- a/nat46/modules/nat46-netdev.c
+++ b/nat46/modules/nat46-netdev.c
@@ -170,6 +170,7 @@ static void nat46_netdev_setup(struct ne
	dev->mtu = 16384; /* iptables does reassembly. Rather than using ETH_DATA_LEN, let's try to get as much mileage as we can with the Linux stack */
	dev->features = NETIF_F_NETNS_LOCAL;
	dev->flags = IFF_NOARP | IFF_POINTOPOINT;
+	dev->priv_flags_ext = IFF_EXT_MAPT;
 }

 int nat46_netdev_create(char *basename, struct net_device **dev)
